js/tests/filters/counterFormatter_filter.js

```diff
@@ -15,7 +15,7 @@ describe('counterFormatter filter', function() {
 		expect(counterFormatter(15)).to.equal(15);
 		expect(counterFormatter(0)).to.equal('');
 		expect(counterFormatter(-5)).to.equal(-5);
-		expect(counterFormatter(999)).to.equal(999);
-		expect(counterFormatter(1000)).to.equal('999+');
+		expect(counterFormatter(9999)).to.equal(9999);
+		expect(counterFormatter(10000)).to.equal('9999+');
```

Comment:
This code just changes the tests. Please also take a look at [this file](https://github.com/nextcloud/contacts/blob/master/js/filters/counterFormatter_filter.js#L7) which contains the actual logic.

Reply:
Many thanks @Henni just realised and changed that also now. :)